technical.md

```diff
@@ -44,17 +44,19 @@
 <details>
 <summary>HuBMAP Data Validation</summary>
 
--   HuBMAP Data Validation is a continuously improving process that starts with data definitions including HuBMAP data QA/QC [standards for donor, sample, tissue](https://portal.hubmapconsortium.org/docs/metadata), and [assay](https://portal.hubmapconsortium.org/docs/assays) being created by a set of corresponding small teams under the Data Coordination Working Group.
+-   HuBMAP Data Validation is a continuously improving process that starts with defining QC/QA standards and establishing definitions for donor, sample and assay metadata. Standards, definitions, metadata schema and data directory schema are created by teams under the Data Coordination Working Group. The metadata schema may be accessed in Github via links from the [metadata page](https://portal.hubmapconsortium.org/docs/metadata) and downloaded as Excel spreadsheets. These spreadsheet templates have dropdown menus of constrained input for certain metadata fields.
```

Comment:
```suggestion
-   HuBMAP Data Validation is a continuously improving process that starts with defining QC/QA standards and establishing definitions for donor, sample and assay metadata.
Standards, definitions, metadata schema and data directory schema are created by teams under the Data Coordination Working Group.
Metadata schemas are available [here](https://hubmapconsortium.github.io/ingest-validation-tools/),
along with Excel templates with dropdowns for data entry.
```
- Suggest adding returns here to keep the diff readable.
- Suggest getting rid of `/docs/metadata`.